xsl/mathbook-html.xsl

```diff
@@ -6084,6 +6084,203 @@ This is a Java Applet created using GeoGebra from www.geogebra.org - it looks li
+                </xsl:otherwise>
+            </xsl:choose>
+        </xsl:attribute>
+        <xsl:attribute name="height">
+            <xsl:choose>
+                <xsl:when test="@height">
+                    <xsl:value-of select="@height"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>400</xsl:text>
+                </xsl:otherwise>
+            </xsl:choose>
+        </xsl:attribute>
+        <xsl:attribute name="scrolling">
+            <xsl:choose>
+                <xsl:when test="@scrolling">
+                    <xsl:value-of select="@scrolling"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>yes</xsl:text>
+                </xsl:otherwise>
+            </xsl:choose>
+        </xsl:attribute>
+        <xsl:attribute name="src">
+            <xsl:text>https://www.geogebra.org/material/iframe/id/</xsl:text>
+            <xsl:value-of select="@geogebratube"/>
+            <xsl:text>/width/</xsl:text>
+            <xsl:choose>
+                <xsl:when test="@width">
+                    <xsl:value-of select="@width"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>400</xsl:text>
+                </xsl:otherwise>
+            </xsl:choose>
+            <xsl:text>/height/</xsl:text>
+            <xsl:choose>
+                <xsl:when test="@height">
+                    <xsl:value-of select="@height"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>400</xsl:text>
+                </xsl:otherwise>
+            </xsl:choose>
+            <xsl:text>/border/</xsl:text>
+            <xsl:choose>
+                <xsl:when test="@border-color">
+                    <xsl:value-of select="@border-color"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>888888</xsl:text>
+                </xsl:otherwise>
+            </xsl:choose>
+            <xsl:text>/smb/</xsl:text>
+            <xsl:choose>
+                <xsl:when test="@menu = 'yes'">
+                    <xsl:text>true</xsl:text>
+                </xsl:when>
+                <xsl:when test="@menu = 'no'">
+                    <xsl:text>false</xsl:text>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>false</xsl:text>
+                </xsl:otherwise>
+            </xsl:choose>
+            <xsl:text>/stb/</xsl:text>
+            <xsl:choose>
+                <xsl:when test="@toolbar = 'yes'">
+                    <xsl:text>true</xsl:text>
+                </xsl:when>
+                <xsl:when test="@toolbar = 'no'">
+                    <xsl:text>false</xsl:text>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>false</xsl:text>
+                </xsl:otherwise>
+            </xsl:choose>
+            <xsl:text>/stbh/</xsl:text>
+            <xsl:if test="@toolbar">
```

Comment:
I think that this `if` should not be here. Like the previous attributes, it should dive into the `choose`.